2023-08-11 19:13:16,274 ERROR: Exception on /edit_profile/clem [POST] [in /Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/Users/danielweir/highlyfe3/app/main/routes.py", line 199, in edit_profile
    return redirect(url_for('main.edit_profile'))
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/helpers.py", line 225, in url_for
    return current_app.url_for(
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1697, in url_for
    return self.handle_url_build_error(error, endpoint, values)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1686, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/werkzeug/routing/map.py", line 950, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'main.edit_profile'. Did you forget to specify values ['username']?
2023-08-11 19:14:01,530 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 01:46:25,686 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 01:52:05,396 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 01:59:21,454 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 02:03:28,623 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:42:01,759 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:42:08,534 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:47:55,713 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:48:59,157 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:50:55,125 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:52:18,367 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:54:11,635 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 11:55:14,090 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:06:16,972 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:07:17,327 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:10:24,974 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:11:03,442 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:11:50,532 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:14:25,688 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:17:31,687 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:18:43,432 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:19:15,119 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:20:15,026 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:20:54,953 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:21:47,766 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 12:22:21,915 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:11:49,175 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:12:11,242 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:15:17,872 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:16:30,347 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:19:28,620 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:20:14,577 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:21:04,286 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:21:43,864 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:21:46,678 ERROR: Exception on /About_Us [POST] [in /Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/Users/danielweir/highlyfe3/app/main/routes.py", line 134, in About_Us
    return render_template('About_Us.html',users=users, title='About Us', name_to_update=name_to_update, id=id, form=form, forms=forms)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/danielweir/highlyfe3/app/templates/About_Us.html", line 120, in template
    {% endblock %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'.
2023-08-12 13:22:44,307 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:23:54,716 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:24:17,355 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:25:05,356 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:25:27,859 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:28:37,036 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:29:03,249 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:29:30,602 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:30:17,437 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:30:33,527 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:32:13,707 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:33:15,403 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:33:36,194 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:34:51,812 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:35:03,853 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:35:16,075 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:35:27,652 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:35:52,177 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:36:27,982 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:38:25,649 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-12 13:38:46,981 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
